app/controllers/admin/users/view/events/list.js

```diff
@@ -1,82 +1,133 @@
-      disableFiltering : true
-    }
-  ],
-  actions: {
-    moveToDetails(id) {
-      this.transitionToRoute('events.view', id);
-    },
-    editEvent(id) {
-      this.transitionToRoute('events.view.edit.basic-details', id);
-    },
-    openDeleteEventModal(id, name) {
-      this.set('isEventDeleteModalOpen', true);
-      this.set('confirmName', '');
-      this.set('eventName', name);
-      this.set('eventId', id);
-    },
-    deleteEvent() {
-      this.set('isLoading', true);
-      this.store.findRecord('event', this.eventId, { backgroundReload: false }).then(function(event) {
-        event.destroyRecord();
-      })
-        .then(() => {
-          this.notify.success(this.l10n.t('Event has been deleted successfully.'));
-        })
-        .catch(() => {
-          this.notify.error(this.l10n.t('An unexpected error has occurred.'));
-        })
-        .finally(() => {
-          this.set('isLoading', false);
-        });
-      this.set('isEventDeleteModalOpen', false);
+export default class extends Controller.extend(EmberTableControllerMixin) {
+
+  @or('authManager.currentUser.isSuperAdmin', 'authManager.currentUser.isAdmin') hasRestorePrivileges;
+
+  @computed()
+  get columns() {
+    return [
+      {
+        name            : 'Name',
+        valuePath       : 'name',
+        isSortable      : true,
+        headerComponent : 'tables/headers/sort',
+        cellComponent   : 'ui-table/cell/cell-event',
+        options         : {
+          hasRestorePrivileges: this.hasRestorePrivileges
+        },
+        actions: {
+          moveToDetails        : this.moveToDetails.bind(this),
+          editEvent            : this.editEvent.bind(this),
+          openDeleteEventModal : this.openDeleteEventModal.bind(this),
+          deleteEvent          : this.deleteEvent.bind(this),
+          restoreEvent         : this.restoreEvent.bind(this)
+        }
+      },
+      {
+        name            : 'Date',
+        valuePath       : 'startsAt',
+        isSortable      : true,
+        headerComponent : 'tables/headers/sort',
+        cellComponent   : 'ui-table/cell/cell-event-date'
+
+      },
+      {
+        name          : 'Sessions',
+        valuePath     : 'sessions',
+        width         : 80,
+        isSortable    : false,
+        cellComponent : 'ui-table/cell/cell-sessions-dashboard'
+      },
+      {
+        name          : 'Speakers',
+        valuePath     : 'speakers',
+        cellComponent : 'ui-table/cell/cell-speakers-dashboard',
+        isSortable    : false
+
+      },
+      {
+        name          : 'Tickets',
+        valuePath     : 'tickets',
+        cellComponent : 'ui-table/cell/cell-tickets',
+        isSortable    : false
+
+      },
+      {
+        name          : 'Public URL',
+        valuePath     : 'url',
+        cellComponent : 'ui-table/cell/cell-link',
+        isSortable    : false
+      }
+    ];
+  }
+
+  @computed('model.[]')
+  get rows() {
```

Comment:
Use model directly. One less computed property. One less iteration of the dataset.